apps/api/v1/test/docker-compose.yml

```diff
@@ -1,6 +1,3 @@
-# Set the version of docker compose to use
-version: "3.9"
-
```

Comment:
Yeah version is obsolete, it now uses the latest Compose Specification by default.